### ISSUE TITLE
Install fail 

### ISSUE THREAD

**Issue by KevinJue** [tomaz/appledoc#588]
Xcode 7.2
OS: El Capitan 10.11.3

I follow readme to start documentation, but on sudo sh install-appledoc.sh a got a install failed.

```
ld: library not found for -lDiscount
clang: error: linker command failed with exit code 1 (use -v to see invocation)

** INSTALL FAILED **


The following build commands failed:
    Ld /tmp/appledoc.dst/usr/local/bin/appledoc normal x86_64
(1 failure)
```

Obviously i install pod and project build correctly on xcode but not with sh, so i dont get the binary exported
It's a bug or i miss something ?

**Comment by tomaz** [comment]
Is this on latest master? There were couple project related changes that might end up with this kind of issue. Also make sure you open appledoc.xcworkspace, not .xcodeproj

**Comment by Coeur** [comment]
same issue... I should I've tried the install script before merging

**Comment by Coeur** [comment]
@KevinJue, can you retry now?

**Comment by KevinJue** [comment]
@tomaz yes it's on master and obviously after pod install i use .xcworkspace.

@Coeur i dont understand, what's do you mean ? what should i do ?

**Comment by KevinJue** [comment]
@tomaz yes it's on master and obviously after pod install i use .xcworkspace.

@Coeur Well i got an other error 

`xcodebuild: error: The workspace named "appledoc" does not contain a scheme named "appledoc". The "-list" option can be used to find the names of the schemes in the workspace.`

file xcworkspace exist.

**Comment by Coeur** [comment]
I'm sorry, the previous fix was gitignored. Now it should be good.

**Comment by KevinJue** [comment]
@Coeur Did you try ? Because actually previous problem is fix but binary doesn't exist. 

Where you doing cp command to move it to /usr/local/bin/, we don't find appledoc.

```
Installing binary to /usr/local/bin/
cp: /tmp/appledoc.dst/usr/local/bin/appledoc: No such file or directory 
```

**Comment by Coeur** [comment]
I got tricked by the bold **\*\* INSTALL SUCCEEDED **** message.

**Comment by Coeur** [comment]
We could add a line to the Build phase "Run Script":

    rm default_templates.zip
    cp $(DSTROOT)$(INSTALL_PATH)/appledoc /usr/local/bin

But then it would hardcode the installation to that path.
We better figure out who to override an INSTALL_DIR of `/var/root/Library/Developer/Xcode/DerivedData/appledoc-fuhaeexfriuyzvbifkmioruvdhie/Build/Intermediates/ArchiveIntermediates/appledoc/InstallationBuildProductsLocation/usr/local/bin`, as `/tmp/appledoc.dst/usr/local/bin` is not correct anymore.

**Comment by Coeur** [comment]
I've found a solution.

**Comment by leechance** [comment]
** INSTALL SUCCEEDED **

install: /tmp/Build/Intermediates/ArchiveIntermediates/appledoc/InstallationBuildProductsLocation/usr/local/bin/appledoc: No such file or directory

i using the 'appledoc --version'  
**-bash: cd: appledoc: No such file or directory**

why ? @Coeur

**Comment by Coeur** [comment]
@leechance you may want to open a new issue, as this one is unlikely to be reopened.

Also, I no longer contribute to appledoc, so @tomaz may help you better than I.

**Comment by AntonyMCs** [comment]
Seems the path is changed (Not sure if in latest Xcode or in OSX 10.13).

You can easily fix it modifiying the script install-appledoc.sh

-install -v /tmp/Build/Intermediates/ArchiveIntermediates/appledoc/InstallationBuildProductsLocation/usr/local/bin/appledoc "$BINARY_DIR"
+install -v /tmp/Build/Intermediates.noindex/ArchiveIntermediates/appledoc/InstallationBuildProductsLocation/usr/local/bin/appledoc "$BINARY_DIR"

At line 45.

Would do a pull request but haven't evaluated the impact on other Xcode/OSX versions so not really safe to just replace for everyone ...

Xcode 9.0
Build version 9A235
OSX 10.13

Br,
Tony

**Comment by Coeur** [comment]
Bah, if you put the details here, I'll reopen.

**Comment by tomaz** [comment]
Please do PR against Xcode 9, it’s latest released version so latest codebase needs to be working against it @AntonyMC

**Comment by AntonyMCs** [comment]
Seems to be fixed, at least with Latest Xcode 9.1:

```
** INSTALL SUCCEEDED **

install: /tmp/Build/Intermediates/ArchiveIntermediates/appledoc/InstallationBuildProductsLocation/usr/local/bin/appledoc -> /usr/local/bin//appledoc
$ xcodebuild -version
Xcode 9.1
Build version 9B55
```

I think this issue can be closed.
